packages/medulas-react-components/package.json

```diff
@@ -18,7 +18,9 @@
     "@material-ui/core": "4.0.2",
     "@material-ui/icons": "4.0.1",
     "@material-ui/styles": "4.0.2",
+    "@types/lodash": "^4.14.134",
     "add": "^2.0.6",
+    "lodash": "^4.17.11",
```

Comment:
Why lodash? This is a huge library and I prefer to do not include it. Where are you using it?

Reply:
hmm, you are right, there is no lodash dependencies.